src/battle_script_commands.c

```diff
@@ -12990,6 +12990,11 @@ static void Cmd_metronome(void)
 
         if (!(sForbiddenMoves[gCurrentMove] & FORBIDDEN_METRONOME))
         {
+            if (gBattleMoves[gCurrentMove].effect == EFFECT_MULTI_HIT)
+                gBattleStruct->atkCancellerTracker = CANCELLER_MULTIHIT_MOVES;
+            else if (gBattleMoves[gCurrentMove].flags & FLAG_POWDER)
+                gBattleStruct->atkCancellerTracker = CANCELLER_POWDER_MOVE;
+
```

Comment:
Shouldn't it go through every move canceller instead of adding these patches?

Reply:
Probably but it goes through them already for other moves. Powder and Multi Hit (except for Scale Shot) moves specifically didn't seem to work.